### ISSUE TITLE
Regular fonts are displayed as Bold.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rakibtg/gdipp#34]
```
Which program triggers the problem? Specify the program filename if
necessary.  NewsBin Pro  Newsbinpro64.exe


What steps will reproduce the problem?
1. Run the program
2.
3.

What is the expected output? What do you see instead?
Text should honor regular or bold setting. Regular fonts are displaying as
bold in some cases.

What version of gdipp are you using? On what operating system (with
bitness)?  gdipp 0.7.4 Beta for 64-bit Windows (English)  
Windows 7 x64.


Please provide any additional information below.
I am getting Bold fonts in the program mentioned above. I've found that
changing the font from Regular to Bold fixes the problem.  It appears
things might be reversed. 


```

Original issue reported on code.google.com by `garys...@gmail.com` on 11 Apr 2010 at 11:37

**Comment by GoogleCodeExporter** [comment]
```
It appears only headings in newsbin are affected by this.  There is one section 
where
the font chosen is used for the headings as well as the data.  Using Regular the
heading is Bold but the data is Regular.  Changing the font to Bold reverses 
this.
```

Original comment by `garys...@gmail.com` on 12 Apr 2010 at 12:04

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
First screen is Bold.  Second screen is Regular.  Both the header and data are 
the
same font.


```

Original comment by `garys...@gmail.com` on 12 Apr 2010 at 12:52

Attachments:
- [Capture.JPG](https://storage.googleapis.com/google-code-attachments/gdipp/issue-34/comment-4/Capture.JPG)
- [Capture1.JPG](https://storage.googleapis.com/google-code-attachments/gdipp/issue-34/comment-4/Capture1.JPG)

**Comment by GoogleCodeExporter** [comment]
```
The heavy bold header is "actually" what Newsbin wants! The weight of such text 
is
956 (weird number). It is possible to be a bug for Newsbin, not gdipp. 
According to
http://msdn.microsoft.com/en-us/library/dd145037%28VS.85%29.aspx, it should be
rendered even heavier than the "Heavy" style.

Since 0.7.4, gdipp has the ability to render ANY weight the program requests. 
It also
means, if the program requests a wrong weight, gdipp renders as-is (with 900
maximum). This feature is not even available in ClearType. ClearType seems only
render text with 400 (regular weight) and 700 (bold weight) (with some 
exceptions
like Segoe UI). Base on my experiments, any weight <= 400 is rendered as 400.
Otherwise, rendered as 700. That's why Newsbin looks OK without gdipp.

Currently there is no solution for this problem from gdipp side. You can send an
email to the Newsbin dev team regarding this.
```

Original comment by `crendk...@gmail.com` on 12 Apr 2010 at 3:42

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Similar problems can be found in Universal Extractor. In the screenshot, the 
"MSI
Installer Extraction" text is rendered in 1200 weight (which is in fact an 
invalid
weight), as requested.

```

Original comment by `crendk...@gmail.com` on 12 Apr 2010 at 3:44
- Added labels: **Priority-Low**, **Type-Other**
- Removed labels: **Priority-Medium**, **Type-Defect**

Attachments:
- [UniExt.png](https://storage.googleapis.com/google-code-attachments/gdipp/issue-34/comment-7/UniExt.png)

**Comment by GoogleCodeExporter** [comment]
```
I will add the weight 900 (HEAVY) limit in the next version. The screenshot
illustrates the difference.
```

Original comment by `crendk...@gmail.com` on 12 Apr 2010 at 3:49

Attachments:
- [UniExt with limit.png](https://storage.googleapis.com/google-code-attachments/gdipp/issue-34/comment-8/UniExt with limit.png)

**Comment by GoogleCodeExporter** [comment]
```
Should be fixed in 0.7.5.
```

Original comment by `crendk...@gmail.com` on 16 Apr 2010 at 7:14

**Comment by GoogleCodeExporter** [comment]
Original comment by `crendk...@gmail.com` on 22 Apr 2010 at 9:22
- Changed state: **Fixed**
